lesson_13/homework_13.py

```diff
@@ -0,0 +1,107 @@
+import csv
+import json
+import xml.etree.ElementTree as ET
+from l_13_csv import read_file, write_csv
+from pathlib import Path
+import logging
+import  os
+
+log_dir = Path(r"C:\Users\qdbp\Documents\Study\aqa_070225\lesson_13\logs")
```

Comment:
```suggestion
log_dir = Path(r"C:\Users\qdbp\Documents\Study\aqa_070225\lesson_13\logs")
```
оце - погано і я пояснював чому